### PR TITLE
T-18302 PHD V1 - Update E2E tests for Relay Meter

### PR DESCRIPTION
**Pull request by commoddity**
The E2E tests for relay meter were no longer working and has to be updated to work with PHD V1 endpoints. I updated them to work with the new test Docker DB image which uses the latest DB schema from the DB driver repo. This has the advantage of simplifying the test code significantly.

### CODE REVIEW

File: api/server.go [hunk 1]
```diff
@@ -101,6 +101,7 @@ func handleOriginClassification(meter RelayMeter, l *logger.Logger, w http.Respo
 }
 
 func handleAppLatency(meter RelayMeter, l *logger.Logger, app string, w http.ResponseWriter, req *http.Request) {
+	fmt.Println("FIRING APP LATENCY")
```
Comment (pablocampogo) [inline api/server.go]: Hi hello, this shouldn't be here

**Review by pablocampogo — DISMISSED** [review]
Small change

File: api/server.go (hunk 1 above)
Comment (commoddity) [reply api/server.go]: ☠️😵☠️

File: cmd/apiserver/main.go [hunk 2]
```diff
@@ -135,11 +136,12 @@ func (b *backendProvider) LoadBalancer(endpoint string) (*types.LoadBalancer, er
 func (b *backendProvider) LoadBalancers() ([]*types.LoadBalancer, error) {
 	// TODO: make the timeout configurable
 	ctx, _ := context.WithDeadline(context.Background(), time.Now().Add(30*time.Second))
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/load_balancer", b.backendApiUrl), nil)
+	fmt.Println("HERE", fmt.Sprintf("%s/load_balancer", b.phdBaseURL))
```
Comment (ricantar) [inline cmd/apiserver/main.go]: Could you remove this please?

**Review by ricantar — APPROVED** [review]
Ty mate

**Pull request merged**